socorro/cron/jobs/clean_raw_adi.py

```diff
@@ -6,29 +10,44 @@
 )
 
 
+SIX_MONTHS = 180
+
+
 @with_postgres_transactions()
 @with_single_postgres_transaction()
 class CleanRawADICronApp(BaseCronApp):
-    app_name = 'clean-raw-adi'
-    app_description = """
+    """Deletes old data from raw_adi and raw_adi_logs
+
     See https://bugzilla.mozilla.org/show_bug.cgi?id=1227131
 
-    See https://mana.mozilla.org/wiki/pages/viewpage.action?pageId=5734601#cra\
-    sh-stats.mozilla.com%28Socorro%29-DataExpirationPolicy
     """
+
+    app_name = 'clean-raw-adi'
+    app_description = 'Delete old data from raw_adi and raw_adi_logs'
+
     required_config = Namespace()
     required_config.add_option(
         'days_to_keep',
-        default=30 * 6,  # rougly 6 months
+        default=SIX_MONTHS,
         doc='Number of days of raw adi to keep in Postgres')
 
     def run(self, connection):
         cursor = connection.cursor()
         # Casting to date because stored procs in psql are strongly typed.
         assert self.config.days_to_keep > 0
+
+        # Clean raw_adi
         cursor.execute(
             """
             DELETE FROM raw_adi
             WHERE date < NOW() - INTERVAL '{} days'
             """.format(self.config.days_to_keep)
         )
+
+        # Clean raw_adi_logs
+        cursor.execute(
+            """
+            DELETE FROM raw_adi_logs
+            WHERE report_date < NOW() - INTERVAL '{} days'
+            """.format(self.config.days_to_keep)
+        )
```

Comment:
I merged this cron job and `CleanRawAdiLogsCronApp` or whatever it was called. They both run fast. They're effectively related. Might as well have one job rather than two.